### ISSUE TITLE
Parse image references from markdown files

### ISSUE THREAD

**Issue by konstin** [axodotdev/oranda#644]
Currently (oranda 0.5.0-prerelease.1), images in the readme need to be in a folder called `static`. It would be great if oranda could instead parse the readme an copy all markdown image inclusions (`![alt](path/to/img.png)`), maybe even html inclusion that you sometimes need to use.

Additionally, it would be nice if oranda would warn about image that are referenced but not present.

**Comment by manyinsects** [comment]
Agreed! Better Markdown parsing (which this would be a part of) is definitely on my list.
